docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -1,41 +1,132 @@
 === macOS Server
 
-This is only required for building iOS applications.
+This is required for building iOS applications. The following specifications are recommended:
+
+* 4GB RAM
+* 2 Cores
+* macOS 10.12 Sierra
+* 30GB minimum disk space and and additional 20GB per XCode version
 
 ==== macOS Machine Setup
 
-- Set `ansible_sudo_pass` variable for the Ansible job to enable the Installer to work with root permissions. It is required by several installation steps.
+Follow the steps below to enable correct configuration of your macOS server via the installation.
+
+
+====== Create SSH User
+
+Ansible requires SSH access to the macOS server in order to correctly provision the machine to support iOS builds. To enable this, it is required that
+a user with sudo permissions and SSH access is created on the target machine. An example bash script is provided to achieve this. 
```

Comment:
I think it's worth mentioning that this user also needs to be added into the admin group.

Reply:
Ok do you think we should make that part of the example script or just mention it here?

Reply:
It already is: `+dseditgroup -o edit -a ${USERNAME} -t user admin` so we just need to mention it.